fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/rule/SingleTableMvRewriteRule.java

```diff
@@ -1,76 +0,0 @@
-// Copyright 2021-present StarRocks, Inc. All rights reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-
-package com.starrocks.sql.optimizer.rule.transformation.materialization.rule;
-
-import com.google.common.collect.Lists;
-import com.starrocks.sql.optimizer.OptExpression;
-import com.starrocks.sql.optimizer.OptimizerContext;
-import com.starrocks.sql.optimizer.OptimizerTraceUtil;
-import com.starrocks.sql.optimizer.operator.OperatorType;
-import com.starrocks.sql.optimizer.operator.pattern.Pattern;
-import com.starrocks.sql.optimizer.rule.Binder;
-import com.starrocks.sql.optimizer.rule.Rule;
-import com.starrocks.sql.optimizer.rule.RuleSetType;
-import com.starrocks.sql.optimizer.rule.RuleType;
-
-import java.util.List;
-
-// used to rewrite query with single table scan pattern to mv
-public class SingleTableMvRewriteRule extends Rule {
-    public SingleTableMvRewriteRule() {
-        super(RuleType.TF_MV_CBO_SINGLE_TABLE_REWRITE_RULE, Pattern.create(OperatorType.PATTERN));
```

Comment:
Why remove this? This is not used, right?

Reply:
its main logic is moved into BestMvSelector. So remove it here